www/pages/getting-started/netlify-cms.md

```diff
@@ -0,0 +1,144 @@
+
+```render javascript
+  render() {
+    return html\`
+      ...
+      <script src="https://identity.netlify.com/v1/netlify-identity-widget.js"></script>
+      ...
+      \`
+  }
+```
+
+> Later, when you're doing this with a your own projects, make sure you either:
+>* a) add this line to all of your custom page templates or
+>* b) add this line to your app template
+
+With that completed, save and recommit all your changes to your repository. If you haven't already pushed your code to a repository on github, you must do so in order to mirror it on netlify.
+
+### Netlify Your Project
+
+
+#### Create New Project
+
+Now we can return to Netlify.com make sure you register, login, and click "new site from git" button.
+
+```render
+<img src="/assets/netlify-create-new.png" alt="netlify-create-new" style="max-width:800px;"/>
+```
+
+Click GitHub and then you must authorize netlify to read your repositories.
+
+Finally, select your project's repository then and the build command: `npm run build` and the publish directory: `public` like this:
+
+```render
+<img src="/assets/netlify-deploy.png" alt="netlify-deploy" style="max-width:700px;"/>
+```
+
+Click **Deploy Site** and you're site will now be deployed to a randomly generated url at netlfy.com
+
+### Configure Authentication
+
+Click **site settings** button, then select **Identity** from the left side menu.
+
+Click the **Enable Identity** button
+
+It's up to you if you want registration to be open/invite only.  Click **Edit Settings** Under the **Registration preferences** subheading to change between open/invite.
+
+```render
+<img src="/assets/netlify-registration.png" alt="netlify-registration" style="max-width:800px;"/>
+```
+
+Under **External Providers** in the drop down menu click "Add Provider" -> "GitHub" and then **Enable Github**
+
+If you chose **invite only** are you registration method, you need to invite yourself and others using the "Identity" page which is linked at the top navigation bar. From here you can click **Invite Users** to invite any email address you need.
+
+```render
+<img src="/assets/netlify-invite.png" alt="netlify-invite" style="max-width:700px;"/>
+```
+
+#### Configure Services
+
+The final step is to **Enable Git Gateway** from the **Services** Menu
+
+Click **Edit Settings** then click "Generate access token in Github"
+
+```render
+<img src="/assets/netlify-git-gateway.png" alt="netlify--git-gateway" style="max-width:800px;"/>
+```
+
+To automatically generate an access token from and for the GitHub API
+
+### Access Netlify CMS
+
+Your site is now deployed and configured, you can now login and access your Netlify CMS from:
+ `https://yourgeneratedurl-ads6387.netlify.com/admin`
+
```

Comment:
Is there anything next add here?  Not sure if you covered an "authoring" content step that would verify everything is working where in the reader would add a test page, see it commit to GitHub, and build in Netlify.  Maybe I just missed it though?

Reply:
No, I didn't add that step because I guess I thought it was self expalanatory. The guide was just to setup netlify cms not using it or testing it out.

Reply:
Yeah, it might be nice to do one of two things to close things out:
1. Provide a couple next steps and walk the reader a couple basic steps

or

1. Share a link with how to get started publishing content with Netlify CMS

Either way, a nice way to tie things up and bring the guide to a nice resolution.  Your call.